src/Drawer/index.js

```diff
@@ -153,13 +157,22 @@ const Drawer = ({
       style={depthStyle}
       className={`drawer drawer--${position} ${
         isOpen && isTransitioning ? `drawer--open--${position}` : ""
-      }`}
+      } ${isVerticalMobileDrawer ? "drawer--vertical--mobile" : ""}`}
```

Comment:
I would recommend `import cc from "classcat"` and use `cc([])` to build the class name. 

https://www.npmjs.com/package/classcat